shared-bindings/wifi/Radio.c

```diff
@@ -186,6 +185,29 @@ MP_PROPERTY_GETSET(wifi_radio_tx_power_obj,
     (mp_obj_t)&wifi_radio_get_tx_power_obj,
     (mp_obj_t)&wifi_radio_set_tx_power_obj);
 
+#if CIRCUITPY_WIFI_RADIO_SETTABLE_LISTEN_INTERVAL
```

Comment:
Don't conditionally enable this here. We want it on all ports. If unsupported, then assigning to it can raise an error using `mp_raise_NotImplementedError()`.